src/ui_parts/tab_bar.gd

```diff
@@ -160,13 +160,13 @@ func _gui_input(event: InputEvent) -> void:
 	
 	queue_redraw()
 	if event is InputEventMouseButton:
+		var hovered_idx := get_hovered_index()
```

Comment:
Can we only define this in the cases where it's used? Fetching the local mouse pos is surprisingly expensive.

Reply:
You're probably practically correct that it doesn't matter in this specific case. Not computing values that won't be used (especially when it's an expensive operation) is just best practice for optimization.